src/useradd.c

```diff
@@ -2432,11 +2433,12 @@ static void create_mail (void)
 		if (NULL == spool) {
 			return;
 		}
-		file = alloca (strlen (prefix) + strlen (spool) + strlen (user_name) + 3);
+		len = strlen (prefix) + strlen (spool) + strlen (user_name) + 3;
```

Comment:
This is not a len, but a size.  I'd make a strong distinction between what strlen(3) gives (a length), and what sizeof() gives (a size).

Reply:
Sorry; I didn't read correctly :(

That was a size, since it's what we're allocating (i.e., it includes the NUL terminators).